src/assets/prosemirror/menu.css

```diff
@@ -0,0 +1,147 @@
+.ProseMirror-textblock-dropdown {
```

Comment:
These vendor styles really aren't used outside the component. Maybe they belong there instead?